projects/plugins/jetpack/jsconfig.json

```
@@ -0,0 +1,12 @@
+{
+	"compilerOptions": {
+		"jsx": "react",
+		"paths": {
+			// fallback to resolving imports relative to _inc/client
+			"*": [ "./*", "./_inc/client/*" ]
```

We should only use the fallback value if it can't be resolved. Thus `*` will remain `*` (default). We will need to add `baseUrl` option and set it to current directory. If we don't do this, paths won't be correctly resolved and the last part of testing instructions (Path suggestion for imports) will fail.

```suggestion
		"baseUrl": ".",
		"paths": {
			// fallback to resolving imports relative to _inc/client
			"*": [ "*", "./_inc/client/*" ]
```